--- theme.js.orig	2023-03-14 02:01:12.221434966 +0100
+++ theme.js	2023-03-14 02:02:45.994098961 +0100
@@ -160,7 +160,6 @@
                         .addClass('current')
                         .attr('aria-expanded','true');
                 }
-                link[0].scrollIntoView();
             }
         }
         catch (err) {
@@ -223,15 +222,15 @@
     return nav;
 };
 
-module.exports.ThemeNav = ThemeNav();
+const themeNav = ThemeNav();
 
 if (typeof(window) != 'undefined') {
     window.SphinxRtdTheme = {
-        Navigation: module.exports.ThemeNav,
+        Navigation: themeNav,
         // TODO remove this once static assets are split up between the theme
         // and Read the Docs. For now, this patches 0.3.0 to be backwards
         // compatible with a pre-0.3.0 layout.html
-        StickyNav: module.exports.ThemeNav,
+        StickyNav: themeNav,
     };
 }
 
